lib/jpegli/decode.h

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) the JPEG XL Project Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
```

Comment:
somewhere here there could be a purpose/why section?

Reply:
Done.